frontend/src/components/pages/HomePage/HomePage.tsx

```diff
@@ -55,15 +55,15 @@ const HomePage = (): React.ReactElement => {
   };
 
   const getLogRecords = async (pageNumber: number) => {
-    const buildingValue = building ? building.value : "";
+    const buildingId = building ? building.value : undefined;
```

Comment:
We may want to filter by multiple ids (similar to what's done below) so this should be:
`const buildingIds = buildings.map((building) => building.value);`